Dockerfile.manifest

```
@@ -11,6 +11,9 @@ ENV DOCKER_PASSWORD $DOCKER_PASSWORD
 ARG DRONE_TAG
 ENV DRONE_TAG $DRONE_TAG
 
+RUN apk update \
+ && apk --no-cache upgrade
```

nit: running `apk update` to update the index cache before running multiple commands that set `--no-cache` doesn't make any sense. You're downloading files that you then tell it to ignore, and are leaving the cache files present in the image - as --no-cache does not remove the APKINDEX files if they exist, it just doesn't load them.
```suggestion
RUN apk --no-cache upgrade
```

---

You are right, Brad. Thanks for the flagging it.